src/Xrefcheck/Verify.hs

```diff
@@ -251,14 +252,26 @@ checkExternalResource :: VerifyConfig
                       -> Text
                       -> IO (VerifyResult VerifyError)
 checkExternalResource VerifyConfig{..} link
+    | isIgnored = return mempty
     | doesReferLocalhost = return mempty
     | otherwise = fmap toVerifyRes $ do
         makeRequest HEAD 0.3 >>= \case
             Right () -> return $ Right ()
             Left   _ -> makeRequest GET 0.7
   where
+    isIgnored =
+        let maybeIsIgnored = (doesMatchAnyRegex link) <$> vcIgnoreRefs
+        in fromMaybe True maybeIsIgnored
```

Comment:
Ah okay, back to that issue: I get all broken external links unnoticed if `ignoreRefs` field is absent. Apparently this code is not exactly what we want.

Reply:
Fixed